odh-model-controller/controllers/inferenceservice_route.go

```diff
@@ -73,6 +73,11 @@ func NewInferenceServiceRoute(inferenceservice *inferenceservicev1.InferenceServ
 			Termination:                   routev1.TLSTerminationReencrypt,
 			InsecureEdgeTerminationPolicy: routev1.InsecureEdgeTerminationPolicyRedirect,
 		}
+	} else {
```

Comment:
The odh model controller code has been moved out of this repo -- we can drop this commit.